docker/nginx/conf.d/client.conf

```diff
@@ -444,6 +450,31 @@ server {
 		proxy_pass http://siad/skynet/skylink/$skylink_v1$path$is_args$args;
 	}
 
+	location @dnslink_lookup {
+		include /etc/nginx/conf.d/include/proxy-buffer;
+
+		set $dnslink '';
+
+		rewrite_by_lua_block {
+			local http = require("socket.http")
+			local ok, statusCode, headers, statusText = http.request {
+				url = "http://dnslink-api:3100/dnslink/" .. ngx.var.host
+			}
+
+			if statusCode == ngx.HTTP_OK then
+				ngx.var.dnslink = headers["skynet-skylink"]
+			else
+				ngx.status = statusCode
```

Comment:
`nit` this is redundant I believe because of `ngx.exit`

Reply:
no, ngx.exit will only exit from the rewrite block but the code you set there is only to indicate whether you want to break the flow (>= 400) or continue it (< 400) - it would return an error message with code 200 if not for this line - I know, really stupid :D

Reply:
lol